charts/opentelemetry-kube-stack/Chart.yaml

```diff
@@ -18,9 +18,12 @@ icon: https://raw.githubusercontent.com/cncf/artwork/a718fa97fffec1b9fd14147682e
 # release, the collector's latest patch will be manually overridden.
 appVersion: 0.120.0
 dependencies:
-  - name: crds
+  - name: openTelemetry-crds
     version: "0.0.0"
-    condition: crds.install
+    condition: crds.openTelemetry
+  - name: monitoringCoreOS-crds
```

Comment:
i don't love the names here, usually you don't see charts with capitalization. we can name them otel-crds and prometheus-crds, as they're not actually exposed externally to this chart.